src/SensitiveFloats.jl

```diff
@@ -84,6 +84,18 @@ function zero_sensitive_float{ParamType <: ParamSet}(
 end
 
 
+function zero_sensitive_float_array{ParamType <: ParamSet}(
+    ::Type{ParamType}, NumType::DataType, local_S::Int, d::Integer...)
+
+    sf_array = Array(SensitiveFloat{ParamType, NumType}, d)
+    for ind in 1:length(sf_array)
+        sf_array[ind] = zero_sensitive_float(ParamType, NumType, local_S)
+    end
+
+    sf_array
+end
```

Comment:
I shouldn't have deleted it and you're planning to use it shortly, so let's get this included. In general, I think we should aim to keep unreachable code (code that doesn't get called from `bin`) out of the code base, and commit it only once it is reachable from `bin`. I mean, there's no need to be too strict about this, but unreachable code is pretty confusing for anyone reading the code who doesn't know it's history, in my experience.